core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -417,6 +417,8 @@
       "The amount of time to sleep between checking for the Montior ZooKeeper lock"),
   MONITOR_LOG_DATE_FORMAT("monitor.log.date.format", "yyyy/MM/dd HH:mm:ss,SSS", PropertyType.STRING, "The SimpleDateFormat string used to configure "
       + "the date shown on the 'Recent Logs' monitor page"),
+  MONITOR_RESOURCES_EXTERNAL("monitor.resources.external", "", PropertyType.STRING,
+      "JSON list of HTML tags of external resources (JS and CSS) to be imported by the Monitor. Be sure to wrap with CDATA tags."),
```

Comment:
I assume the CDATA tag wrapping necessary to avoid the values being interpreted as XML?
Description could mention that if this is not set, then, the monitor will try to load external resources, such as jquery, bootstrap, etc., from its class path.
We could put the default ones here instead of handling that in the template, but it'd be ugly. I'm fine with doing it either way, but the description should be updated to mention where it's loading them from when it's not set.

Reply:
I will add more tips to the description and location of the default values